dom-querying/dom-query-quiz-notes.md

```diff
@@ -7,7 +7,7 @@ After completing this exercise, you should be able to discuss or answer the foll
 - Why do we log things to the console?
   to test scripts
 - What is a "model"?
-
+  a model is a replica of the html
```

Comment:
It's moreso asking in general, so more like just a "recreation of something". In the case of the DOM, it's a recreation of our HTML page, but in general, it just refers to a recreation of something